libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -34,17 +39,23 @@ ControlFlowBuilder::ControlFlowBuilder(CFG::NodeContainer& _nodeContainer, Funct
 {
 }
 
-
-unique_ptr<FunctionFlow> ControlFlowBuilder::createFunctionFlow(
-	CFG::NodeContainer& _nodeContainer,
-	FunctionDefinition const& _function
-)
+std::unique_ptr<FunctionFlow> ControlFlowBuilder::initFunctionFlow(CFG::NodeContainer& _nodeContainer)
```

Comment:
Does this need to be a member function?

Reply:
It's actually a static member function, but it could be a completely free/local function. Would you prefer that?

Reply:
Yes